pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -75,6 +75,11 @@ with stdenv.lib;
   ''}
   INTEL_IDLE y
   CPU_FREQ_DEFAULT_GOV_PERFORMANCE y
+  ${optionalString (versionAtLeast version "4.9") ''
+    CPU_FREQ_GOV_SCHEDUTIL y
```

Comment:
@NeQuissimus 
Since its been almost a month without feedback from other people here, I think it would be fine to just allow people to use schedutil but not change the default governor yet. Currently no one can use this.

Reply:
I agree with worldofpeace. This should probably be split into two pull requests, one allowing users to use schedutil in the first place, and another that suggests changing he defaults.
That will also allow people to test things out, and / or draw their attention to the fact, that this exists at all.